ios/browser/ui/webui/ai_chat/DEPS

```diff
@@ -0,0 +1,6 @@
+include_rules = [
+  "+brave/components/ai_chat/core/common",
+  "+brave/components/ai_chat/core/common/mojom",
```

Comment:
Do we need both?

Reply:
You're right we dont need both, I'll remove the mojom entry